v63003/outref/gtm8587.txt

```diff
@@ -0,0 +1,10 @@
+# RUNNING TEST FOR $KEY
```

Comment:
I would like test cases that show $key as null and as non-null (like the release note states).